app/controllers/candidate_interface/candidate_interface_controller.rb

```diff
@@ -16,6 +17,10 @@ def add_identity_to_log(candidate_id = current_candidate&.id)
       Raven.extra_context(application_support_url: support_interface_application_form_url(current_application))
     end
 
+    def check_cookie_preferences
```

Comment:
`setup_google_analytics` might be more descriptive?

Reply:
I think cookie_preferences is a better name, as analytics is part of it and cookies can easily be extended to include other properties.